ci/edmtool.py

```diff
@@ -106,7 +106,6 @@
     "traitsui",
     "pyface",
     "pypdf2",
-    "swig",
```

Comment:
i'm not sure why you removed `swig` from the dependencies list here. Without this, `swig` won't get installed via edm into the python environment.

Reply:
it was listed twice, see line 104, I should have mentioned that

Reply:
ahh. i should have noticed that things were not in alphabetical order - which i what i was kinda expecting.

can you rearrange the set alphabetically, for readability? this could be done in a later PR